cadence/contracts/Recipe.cdc

```diff
@@ -0,0 +1,33 @@
+import "ExampleNFT"
+import "NonFungibleToken"
+
+access(all) contract Recipe {
+    access(all) let ExampleNFTMinterPath: StoragePath
+
+    init() {
+        // Reference the storage path where ExampleNFT stores its NFTMinter
+        self.ExampleNFTMinterPath = /storage/exampleNFTMinter
+    }
+
+    access(all) fun mintNFT(
```

Comment:
In the way this recipe was set up, the `Recipe` code's `mintNFT` function just calls the `ExampleNFT` function under the hood. This seems to be duplicating code that already existed in the example transaction. We could potentially replace `Recipe` with the `ExampleNFT` itself (we have this paradigm in a few of the other repos), although it comes with a lot of auxiliary code that is not directly related to the minting function @franklywatson

Reply:
Going ahead with this change given previous repo comments